examples/fabric-admin/scripts/run_fabric_source.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Default paths
+FABRIC_ADMIN_PATH="./fabric-admin"
+FABRIC_BRIDGE_APP_PATH="./fabric-bridge-app"
+FABRIC_ADMIN_LOG="/tmp/fabric_admin.log"
+FABRIC_BRIDGE_APP_LOG="/tmp/fabric_bridge_app.log"
+
+# Detect platform and set default paths
+PLATFORM=$(uname -m)
+if [[ "$PLATFORM" == "x86_64" ]]; then
+    FABRIC_ADMIN_PATH="out/debug/standalone/fabric-admin"
+    FABRIC_BRIDGE_APP_PATH="out/debug/standalone/fabric-bridge-app"
+fi
+
+# Check if --verbose option is specified
+VERBOSE=false
+for arg in "$@"; do
+    if [ "$arg" == "--verbose" ]; then
+        VERBOSE=true
+        break
+    fi
+done
+
+# Stop any running instances and clean up
+
+if [[ "$PLATFORM" == "x86_64" ]]; then
```

Comment:
can we get the path to the current script and use that as a relative location? It seems we have these 3 scripts together always.